src/app/bot.ts

```diff
@@ -36,18 +36,33 @@ export class Bot {
   private async _registerPlugins(): Promise<void> {
     this.container.pluginService.reset();
 
-    try {
-      const pluginExtension =
-        Environment.Playground === Mode.Production ? '.plugin.js' : '.plugin.ts';
-      const files = (await fs.readdir(path.join(__dirname, './plugins'))) || [];
-
-      files
-        .filter((file) => file.endsWith(pluginExtension))
-        .map((plugin) => plugin.replace(pluginExtension, ''))
-        .forEach((plugin) => this.container.pluginService.register(plugin, this.container));
-    } catch (e) {
-      this.container.loggerService.error(e);
-    }
+    const pluginFolder = './src/app/plugins';
+    fs.readdir(pluginFolder, (_err, files) => {
+      files.forEach(async file => {
+
+        // Import the class from the plugin file.
+        const thing = await import(`./plugins/${file}`);
```

Comment:
can we use a more descriptive name? maybe `pluginImport` or rename what `plugin` is known as now to `pluginInstance` and move `thing` to be = `plugin`

Reply:
👀